components/LoginModal.vue

```diff
@@ -168,12 +161,22 @@ export default {
     SfLoader,
   },
   setup() {
+    const { user, login, loading, error: userError, isAuthenticated } = useUser()
     const { isLoginModalOpen, toggleLoginModal } = useUiState()
     const form = ref({})
     const isLogin = ref(false)
     const isForgotten = ref(false)
     const rememberMe = ref(false)
     const createAccount = ref(false)
+    const error = reactive({
```

Comment:
Why create a new reactive reference for error?
Can we just use userError?

Reply:
we can use useError as well.